libriscv.a not mach-o or llvm bitcode file 'libspike_main.a' for architecture arm64

Using "brew install riscv-tools". Is this because of Xcode 14.0?
os:  macOS 12.6
MacBook Air M2
```
==> Installing riscv/riscv/riscv-tools dependency: riscv-isa-sim
==> ../configure --prefix=/opt/homebrew/Cellar/riscv-isa-sim/main --with-target=../../riscv-pk/main/riscv64-unknown-elf --without-boost --without-boost-asio --without-boost-regex
==> make
Last 15 lines from /Users/grejioh/Library/Logs/Homebrew/riscv-isa-sim/02.make:
make: *** Waiting for unfinished jobs....
ld: in libspike_main.a(libriscv.a), archive member 'libriscv.a' with length 12316840 is not mach-o or llvm bitcode file 'libspike_main.a' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: in libspike_main.a(libriscv.a), archive member 'libriscv.a' with length 12316840 is not mach-o or llvm bitcode file 'libspike_main.a' for architecture arm64
make: *** [termios-xspike] Error 1
make: *** [xspike] Error 1
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [spike] Error 1
ld: in libspike_main.a(libriscv.a), archive member 'libriscv.a' with length 12316840 is not mach-o or llvm bitcode file 'libspike_main.a' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libcustomext.so] Error 1
ld: in libspike_dasm.a(libriscv.a), archive member 'libriscv.a' with length 12316840 is not mach-o or llvm bitcode file 'libspike_dasm.a' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [spike-dasm] Error 1
```

--- a/Makefile.in
+++ b/Makefile.in
@@ -241,7 +241,7 @@ $(2)_lib_libnames   := $$(patsubst %, lib%.a, $$($(2)_lib_libs))
 $(2)_lib_libarg     := $$(patsubst %, -l%, $$($(2)_lib_libs))
 $(2)_lib_libnames_shared	:= $$(if $$($(2)_install_shared_lib),lib$(1).so,)
 
-lib$(1).a : $$($(2)_objs) $$($(2)_c_objs) $$($(2)_lib_libnames)
+lib$(1).a : $$($(2)_objs) $$($(2)_c_objs)
 	$(AR) rcs $$@ $$^
 lib$(1).so : $$($(2)_objs) $$($(2)_c_objs) $$($(2)_lib_libnames_shared) $$($(2)_lib_libnames)
 	$(LINK) -shared -o $$@ $(if $(filter Darwin,$(shell uname -s)),-install_name $(install_libs_dir)/$$@) $$^ $$($(2)_lib_libnames) $(LIBS)

